watertap/examples/flowsheets/RO_multiperiod_model/pricetaker_multiperiod_RO.py

```diff
@@ -209,28 +210,37 @@ def visualize_results(m, t_blocks, data):
     ax[0, 0].set_xlabel("Time [hr]")
     ax[0, 0].set_ylabel("Electricity price [$/kWh]")
 
-    ax[1, 0].plot(time_step, pump1_flow * 3600, label="P1")
-    ax[1, 0].plot(time_step, pump2_flow * 3600, label="P2")
+    ax[1, 0].plot(time_step, 100 * pump1_flow / max(pump1_flow), label="Main pump")
+    ax[1, 0].plot(time_step, 100 * pump2_flow / max(pump2_flow), label="Boost pump")
     ax[1, 0].set_xlabel("Time [hr]")
-    ax[1, 0].set_ylabel("Pump flowrate [m3/hr]")
+    ax[1, 0].set_ylabel("Pump flowrate [% of max]")
+    ax[1, 0].set_ylim([0, 100])
 
     ax[2, 0].plot(time_step, recovery * 100)
     ax[2, 0].set_xlabel("Time [hr]")
     ax[2, 0].set_ylabel("Water Recovery [%]")
+    ax[2, 0].set_ylim([1, 100])
 
-    ax[0, 1].plot(time_step, power)
+    ax[0, 1].plot(time_step, 100 * power / max(power))
     ax[0, 1].set_xlabel("Time [hr]")
-    ax[0, 1].set_ylabel("Net Power [kWh]")
+    ax[0, 1].set_ylabel("Net Power [% of max power]")
+    ax[0, 1].set_ylim([0, 100])
 
-    ax[1, 1].plot(time_step, pump1_efficiency * 100, label="P1")
-    ax[1, 1].plot(time_step, pump2_efficiency * 100, label="P2")
+    ax[1, 1].plot(
+        time_step, 100 * pump1_efficiency / max(pump1_efficiency), label="Main pump"
+    )
+    ax[1, 1].plot(
+        time_step, 100 * pump2_efficiency / max(pump2_efficiency), label="Boost pump"
+    )
     ax[1, 1].set_xlabel("Time [hr]")
-    ax[1, 1].set_ylabel("Pump efficiency [%]")
+    ax[1, 1].set_ylabel("Pump efficiency [% of max efficiency]")
     ax[1, 1].legend()
+    ax[1, 1].set_ylim([0, 100])
 
-    ax[2, 1].plot(time_step, pressure * 1e-5)
+    ax[2, 1].plot(time_step, 100 * pressure / (80e5))
```

Comment:
@arao53  With regard to the pressure, now that I think about it, the capital cost of the pump will vary by pressure and flow rate (essentially power consumption) unless we specify that the pump should be costed by the design pressure and flow rate of the pump. Not sure if that's the case